ckanext/privatedatasets/templates_2.8/package/search.html

```diff
@@ -12,7 +12,7 @@
         <a href="{{ h.url_for('privatedatasets.acquired_datasets') }}" title="{{ notifications_tooltip }}"
         class="btn btn-primary">
             <i class="fa fa-archive" aria-hidden="true"></i>
-            <span class="text">{{ _('See acquired datasets list') }}</span>
+            <span class="text">{{ _('Datasets Shared with me') }}</span>
```

Comment:
Just to make it title case:
```suggestion
            <span class="text">{{ _('Datasets Shared With Me') }}</span>
```